### ISSUE TITLE
Truncated mvn sampler in torch??

### ISSUE THREAD

**Issue by HojunYou** [brunzema/truncated-mvn-sampler#2]
I've been looking for a truncated mvn sampler in pytorch and I found your github. It's been a great help, but I'm having hard time  converting this script into pytorch with reparametrization. It's very unlikely, but do you have a plan to convert this into pytorch??

**Comment by brunzema** [comment]
Hi,
i was not planning on converting it to PyTorch, but might do so if I have some spare time.
What is the issue with using Numpy for the sampling?
My other idea was to convert it to Numba for super fast sampling. Would that be maybe of more interest?

Cheers

**Comment by HojunYou** [comment]
Thanks for the reply.

I'm trying to model parameters in the truncated mvn sampler (mean, sigma) by neural networks and learn those parameters via a reinforcement learning framework. So, I need a truncated mvn sampler that can handle gradients (reparametrization), which is why I need it in torch. But, I believe that's my job to do. I asked you just in case. 

Anyway, nice work and thank you!
